tools/pcli/main.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/go-ssz"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/state"
+	stateTrie "github.com/prysmaticlabs/prysm/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/beacon-chain/state/stateutil"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/version"
+	log "github.com/sirupsen/logrus"
+	prefixed "github.com/x-cray/logrus-prefixed-formatter"
+	"gopkg.in/d4l3k/messagediff.v1"
+	"gopkg.in/urfave/cli.v2"
+)
+
+func main() {
+	var blockPath string
+	var preStatePath string
+	var expectedPostStatePath string
+
+	customFormatter := new(prefixed.TextFormatter)
+	customFormatter.TimestampFormat = "2006-01-02 15:04:05"
+	customFormatter.FullTimestamp = true
+	log.SetFormatter(customFormatter)
+
+	app := cli.App{}
+	app.Name = "pcli"
+	app.Usage = "A command line utility to run eth2 specific commands"
+	app.Version = version.GetVersion()
+	app.Commands = []*cli.Command{{
+		Name:     "state-transition",
+		Category: "state-transition",
+		Usage:    "Subcommand to run manual state transitions",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "blockPath",
+				Usage:       "Path to block file(ssz)",
+				Destination: &blockPath,
+			},
+			&cli.StringFlag{
+				Name:        "preStatePath",
```

Comment:
our cli flags are using dashes usually
```suggestion
				Name:        "pre-state-path",
```